src/status_im2/contexts/chat/messages/content/text/style.cljs

```diff
@@ -20,6 +23,30 @@
    :padding-left      10
    :border-left-color colors/neutral-40})
 
+(def mention-tag-wrapper
+  {:position           :relative
+   :padding-vertical   0
+   :padding-horizontal 0
+   :margin-vertical    0
+   :margin-horizontal  0
+   :flex-direction     :row
+   :align-items        :center})
+
+(def mention-tag
+  {:background-color    colors/primary-50-opa-10
+   :position            :relative
+   :padding-horizontal  3
+   :border-radius       6
+   :margin-bottom       -3
+   :z-index             1})
+
+(def mention-tag-text
```

Comment:
Here we should use `defn` instead of `def` because `colors/theme-colors` rely on a Reagent atom to pick the color based on the current theme. With `def`, if the user switches theme the value of `mention-tag-text map won't be redefined.

Reply:
Sure, I used `defn` here instead of `def`.